src/components/submission/basic-details.vue

```diff
@@ -45,13 +45,13 @@ except according to the terms contained in the LICENSE file.
         <div v-if="submission.__system.deviceId != null">
           <dt>{{ $t('deviceId') }}</dt>
           <dd>
-            <span :title="submission.__system.deviceId">{{ submission.__system.deviceId }}</span>
+            <span v-tooltip.text>{{ submission.__system.deviceId }}</span>
           </dd>
         </div>
         <div v-if="submissionVersion.userAgent != null">
           <dt>{{ $t('userAgent') }}</dt>
           <dd id="submission-basic-details-user-agent">
-            <span :title="submissionVersion.userAgent">{{ submissionVersion.userAgent }}</span>
+            {{ submissionVersion.userAgent }}
```

Comment:
We actually don't need a tooltip here because we added word-wrap to this specific element.